### ISSUE TITLE
Problems in PNN

### ISSUE THREAD

**Issue by Drone-Banks** [rixwew/pytorch-fm#5]
Paper said :
> The first hidden layer is fully connected with the product layer. The inputs to it consist of linear
signals lz and quadratic signals lp. With respect to lz and lp inputs, separately, the formulation of l1 is: l1 = relu(lz + lp + b1),

But in your code, the linear signals and quadratic signals are concated :
```
embed_x = self.embedding(x)
cross_term = self.pn(embed_x)
x = torch.cat([embed_x.view(-1, self.embed_output_dim), cross_term], dim=1)
x = self.mlp(x)
return torch.sigmoid(x.squeeze(1))
```

**Comment by Drone-Banks** [comment]
I guess that you had confused  the output layers of PNN and DCN. @rixwew

**Comment by rixwew** [comment]
Thank you for your comment! I'll check it.

**Comment by Drone-Banks** [comment]
> Thank you for your comment! I'll check it.

: )  and I created a pull request

**Comment by rixwew** [comment]
I merged your pr #7 
Thank you:)
